<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InspectorHairpin</class>
 <widget class="QWidget" name="InspectorHairpin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>391</width>
    <height>363</height>
   </rect>
  </property>
  <property name="accessibleName">
   <string>Hairpin Inspector</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QToolButton" name="title">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Hairpin</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::HLine</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="lineWidth">
      <number>2</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="panel" native="true">
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>3</number>
      </property>
      <property name="rightMargin">
       <number>3</number>
      </property>
      <property name="horizontalSpacing">
       <number>3</number>
      </property>
      <property name="verticalSpacing">
       <number>0</number>
      </property>
      <item row="10" column="0">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Begin text:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy">
         <cstring>beginText</cstring>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QSpinBox" name="veloChange">
        <property name="accessibleName">
         <string>Velocity change</string>
        </property>
        <property name="maximum">
         <number>127</number>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Velocity change:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy">
         <cstring>veloChange</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="hairpinType">
        <property name="accessibleName">
         <string>Type</string>
        </property>
        <item>
         <property name="text">
          <string>Crescendo</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Decrescendo</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Dynamic range:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy">
         <cstring>dynRange</cstring>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Continue height:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy">
         <cstring>hairpinContHeight</cstring>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Height:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy">
         <cstring>hairpinHeight</cstring>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QDoubleSpinBox" name="hairpinHeight">
        <property name="accessibleName">
         <string>Height</string>
        </property>
        <property name="suffix">
         <string>sp</string>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Type:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy">
         <cstring>hairpinType</cstring>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QDoubleSpinBox" name="hairpinContHeight">
        <property name="accessibleName">
         <string>Continue height</string>
        </property>
        <property name="suffix">
         <string>sp</string>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="dynRange">
        <property name="accessibleName">
         <string>Dynamic range</string>
        </property>
        <item>
         <property name="text">
          <string>Staff</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Part</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>System</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Placement:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy">
         <cstring>placement</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="placement">
        <property name="accessibleName">
         <string>Placement</string>
        </property>
        <item>
         <property name="text">
          <string>Above</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Below</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QCheckBox" name="hairpinCircledTip">
        <property name="focusPolicy">
         <enum>Qt::TabFocus</enum>
        </property>
        <property name="text">
         <string>Circled tip</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Font face:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy">
         <cstring>fontFace</cstring>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Style:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QSpinBox" name="fontSize">
        <property name="accessibleName">
         <string>Font size</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QFontComboBox" name="fontFace">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>150</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="accessibleName">
         <string>Font face</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="label_8">
        <property name="accessibleName">
         <string>Font size</string>
        </property>
        <property name="text">
         <string>Font size:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy">
         <cstring>fontSize</cstring>
        </property>
       </widget>
      </item>
      <item row="11" column="0">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>End text:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy">
         <cstring>endText</cstring>
        </property>
       </widget>
      </item>
      <item row="11" column="1">
       <widget class="QLineEdit" name="endText">
        <property name="accessibleName">
         <string>End text</string>
        </property>
       </widget>
      </item>
      <item row="10" column="1">
       <widget class="QLineEdit" name="beginText">
        <property name="accessibleName">
         <string>Begin text</string>
        </property>
       </widget>
      </item>
      <item row="9" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="Ms::FontStyleSelect" name="fontStyle"/>
        </item>
       </layout>
      </item>
      <item row="0" column="2">
       <widget class="Ms::ResetButton" name="resetHairpinCircledTip"/>
      </item>
      <item row="2" column="2">
       <widget class="Ms::ResetButton" name="resetPlacement"/>
      </item>
      <item row="3" column="2">
       <widget class="Ms::ResetButton" name="resetDynRange"/>
      </item>
      <item row="4" column="2">
       <widget class="Ms::ResetButton" name="resetVeloChange"/>
      </item>
      <item row="5" column="2">
       <widget class="Ms::ResetButton" name="resetHairpinHeight"/>
      </item>
      <item row="6" column="2">
       <widget class="Ms::ResetButton" name="resetHairpinContHeight"/>
      </item>
      <item row="7" column="2">
       <widget class="Ms::ResetButton" name="resetFontFace"/>
      </item>
      <item row="8" column="2">
       <widget class="Ms::ResetButton" name="resetFontSize"/>
      </item>
      <item row="9" column="2">
       <widget class="Ms::ResetButton" name="resetFontStyle"/>
      </item>
      <item row="10" column="2">
       <widget class="Ms::ResetButton" name="resetBeginText"/>
      </item>
      <item row="11" column="2">
       <widget class="Ms::ResetButton" name="resetEndText"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Ms::FontStyleSelect</class>
   <extends>QWidget</extends>
   <header>inspector/fontStyleSelect.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Ms::ResetButton</class>
   <extends>QWidget</extends>
   <header>inspector/resetButton.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>title</tabstop>
  <tabstop>hairpinCircledTip</tabstop>
  <tabstop>hairpinType</tabstop>
  <tabstop>placement</tabstop>
  <tabstop>dynRange</tabstop>
  <tabstop>veloChange</tabstop>
  <tabstop>hairpinHeight</tabstop>
  <tabstop>hairpinContHeight</tabstop>
  <tabstop>fontFace</tabstop>
  <tabstop>fontSize</tabstop>
  <tabstop>beginText</tabstop>
  <tabstop>endText</tabstop>
 </tabstops>
 <resources>
  <include location="../musescore.qrc"/>
 </resources>
 <connections/>
</ui>
